tests/failpoints/cases/test_transaction.rs

```diff
@@ -68,8 +71,14 @@ fn test_atomic_getting_max_ts_and_storing_memory_lock() {
         .unwrap();
 
     let (prewrite_tx, prewrite_rx) = channel();
+    let (fp_tx, fp_rx) = sync_channel(1);
     // sleep a while between getting max ts and store the lock in memory
-    fail::cfg("before-set-lock-in-memory", "sleep(500)").unwrap();
+    fail::cfg_callback("before-set-lock-in-memory", move || {
+        thread::sleep(Duration::from_millis(200));
```

Comment:
What's the purpose of this sleep before sending to channel?

Reply:
It's now unncessary. I've removed it.